packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -114,6 +115,37 @@ function ActionsMenu({ patientUuid }: { patientUuid: string }) {
   );
 }
 
+function EditMenu({
+  patientUuid,
+  queueUuid,
+  queueEntryUuid,
+}: {
+  patientUuid: string;
+  queueEntryUuid: string;
+  queueUuid: string;
+}) {
+  const { t } = useTranslation();
+  const launchEditPriorityModal = useCallback(() => {
```

Comment:
I think you can change the naming of the modal here to reflect what we are editing, is it a priority modal we are launching or status modal or is it for both priority and status?

Reply:
Done